### PR TITLE
vdo: document versions that have read cache options

### PR DESCRIPTION
**Pull request by bgurney-rh**
The "readcache" and "readcachesize" options, which correspond to
the same options in the "vdo create" and "vdo modify" commands,
have been removed in VDO version 6.2.  Document that the read
cache options are only in VDO 6.1 and older.

##### SUMMARY

The read cache feature in VDO has been removed in VDO version 6.2.  No code needs to be changed; instead, add documentation lines to the "readcache" and "readcachesize" options, stating that they are available in VDO versions 6.1 and older, allowing those versions to continue to use playbooks with those options.

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

vdo

##### ADDITIONAL INFORMATION

If playbooks for systems with VDO 6.2 and newer do not specify the "readcache" or "readcachesize" options, there should not be any errors.  If either of these options are specified for VDO 6.2 and above, there will be an error from the "vdo" command stating "unrecognized arguments: --readCache..." and/or "unrecognized arguments: --readCache..."; these errors should be passed through by ansible, and visible to the user.

```paste below

```

### CODE REVIEW

File: lib/ansible/modules/system/vdo.py [hunk 1]
```diff
@@ -127,6 +127,8 @@
               volumes will maintain their previously configured
               setting unless a different value is specified in the
               playbook.
+              The read cache feature is available in VDO 6.1 and
```
Comment (acozine) [inline lib/ansible/modules/system/vdo.py]:
```suggestion
            - The read cache feature is available in VDO 6.1 and older.
```

File: lib/ansible/modules/system/vdo.py [hunk 2]
```diff
@@ -127,6 +127,8 @@
...
+              older.
```
Comment (acozine) [inline lib/ansible/modules/system/vdo.py]:
```suggestion
```

File: lib/ansible/modules/system/vdo.py [hunk 3]
```diff
@@ -142,6 +144,8 @@
               Existing volumes will maintain their previously
               configured setting unless a different value is specified
               in the playbook.
+              The read cache feature is available in VDO 6.1 and
```
Comment (acozine) [inline lib/ansible/modules/system/vdo.py]:
```suggestion
           -  The read cache feature is available in VDO 6.1 and older.
```

File: lib/ansible/modules/system/vdo.py [hunk 4]
```diff
@@ -142,6 +144,8 @@
...
+              older.
```
Comment (acozine) [inline lib/ansible/modules/system/vdo.py]:
```suggestion
```

**Review by acozine — COMMENTED** [review]
Prefer to call this out on a single, separate line for easier readability.

**Comment by bgurney-rh** [comment]
My repo is set up incorrectly; I'll abandon this PR and try again later.

**Pull request closed without merging**